### DISCUSSION TITLE
possibility of live stream recording?

### DISCUSSION THREAD

**Discussion by RonnieBlaze** [UltimaHoarder/UltimaScraper#905] [Ideas]
would it be possible to add live stream recording?

**Comment by elephanticefire** [comment]
Well it's not as straight forward as scraping images and videos off of a profile. The streams appear to be handled by webrtc. I've been working on it off and on for quite some time now but I honestly have nothing to show for it. I'm beginning to wonder if it is even possible at all. 

Has anyone else dabbled with this?
